controllers/__init__.py

```diff
@@ -1,3 +1 @@
-# -*- coding: utf-8 -*-
-
-from . import controllers
\ No newline at end of file
+from . import controllers
```

Comment:
Remove this file also remove the controllers folder. When you need this you'll add it.